templates/careers/progression.html

```diff
@@ -1,32 +1,343 @@
+            <span class="u-off-screen">Next slides</span>
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="u-hide--medium u-hide--small">
+    {% include 'careers/_careers-progression-slides.html' %}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--careers-progression">
+  <div class="u-fixed-width" data-animation="u-animation--slide-from-bottom">
+    <p>Our mission in action</p>    
+  <h3 class="p-heading--2 p-strip--careers-progression u-no-padding--top">See what's new at Canonical</h3>
+  </div>
+  <div class="row u-extra-space" data-animation="u-animation--slide-from-bottom">
+    <div class="col-6 col-medium-3">
+      {{
+        image (
+        url="https://assets.ubuntu.com/v1/646204f1-Elektrobit 1.png",
+        alt="EB corbos Linux with Elektrobit",
+        width="633",
+        height="435",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+    <div class="col-6 col-medium-3">
+      <small>21 February 2023</small>
+      <h3>Elektrobit and Canonical announces EB corbos Linux &dash; built on Ubuntu</h3>
+      <small><a href="https://ubuntu.com/blog/elektrobit-and-canonical-announce-eb-corbos-linux-built-on-ubuntu">Read more&nbsp;&rsaquo;</a></small>
+    </div>
+  </div>
+  <div class="row u-extra-space" data-animation="u-animation--slide-from-bottom">
+    <div class="col-6 col-medium-3">
+      {{
+        image (
+        url="https://assets.ubuntu.com/v1/058ff4a7-nvidia.png",
+        alt="Ubuntu core",
+        width="633",
+        height="453",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+    <div class="col-6 col-medium-3">
+      <small>20 September 2023</small>
+      <h3>Ubuntu Core set to redefine industrial computing with new edge AI platform NVIDIA IGX</h3>
+      <small><a href="https://ubuntu.com/blog/ubuntu-core-set-to-redefine-industrial-computing-with-new-edge-ai-platform-nvidia-igx">Read more&nbsp;&rsaquo;</a></small>
+    </div>
+  </div>
+  <div class="row u-extra-space" data-animation="u-animation--slide-from-bottom">
+    <div class="col-6 col-medium-3">
+      {{
+        image (
+        url="https://assets.ubuntu.com/v1/cc2b45fe-Virgin Media O2 1.png",
+        alt="",
+        width="633",
+        height="435",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+    <div class="col-6 col-medium-3">
+      <small>7 December 2022</small>
+      <h3>Virgin Media O2 deploys Charmed OSM to accelerate network function virtualisation for cloud workloads</h3>
+      <small><a href="https://ubuntu.com/blog/virgin-media-o2-deploys-charmed-osm-to-accelerate-network-functions-virtualisation-for-cloud-workloads">Read more&nbsp;&rsaquo;</a></small>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>What team members say</h2>
+  </div>
+</section>
```

Comment:
Could you merge this section with the one below somehow as they are connected content?